resources/views/entities/components/posts.blade.php

```diff
@@ -37,7 +37,9 @@
             @include('entities.components.entry')
             @php $entryShown = true @endphp
         @endif
-
+        @if ($post->layout && !$campaign->superboosted())
+            @continue
+        @endif
         @include('entities.components._post')
```

Comment:
This is where to add the new includeWhen logic, and keep _post as it was before (revert all changes)